CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Winetricks
+
+## License
+
+Winetricks is licensed under the LGPL 2.1 or later. Sending a pull request indicates your willingness to license your contribution under this license.
+
+## Coding standards
+* Documented at top of ```src/winetricks```
+* POSIX shell, no bash (maintainer's scripts can be python/bash)
+* If something is unclear, ask
+
+## Making a patch:
+* Check out the source: ```git clone git@github.com:Winetricks/winetricks.git```
+* Hack the source: ```vi src/winetricks```
+* Test it: ```./src/winetricks -q -v foo```
```

Comment:
You might want to mention that it should be tested against the latest version of Wine.

Reply:
Ideally, winetricks isn't tied to a particular wine version (that's why there is w_workaround_wine_bug and friends), but yeah, a comment explaining more is a good idea, thanks.